AnnoyancesFilter/sections/popups.txt

```diff
@@ -112,6 +112,9 @@ bankbazaar.com#$#.oneTapLoginModal[style] { display: none !important; }
 bankbazaar.com#$#.modal-backdrop { display: none !important; }
 bankbazaar.com#$##js-sign-in-box[style="display: block;"] ~ .modal-backdrop { display: block !important; }
 magadanmedia.ru###social_modalblock
+inc42.com#$#iframe[id^="offer-"] { display: none !important }
+inc42.com#$#.tp-backdrop { display: none !important; }
+inc42.com#$#body { overflow: visible !important; }
```

Comment:
It seems that it doesn't work with apps, because there is `!important` already and it has higher priority.
Please change it to something like:
`inc42.com#$#body.tp-modal-open { overflow: visible !important; }`